README.md

```diff
@@ -30,3 +30,7 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+When going through the loops the first one is n* n , the second is just n and the third is also n*n so the total time for the loops is $(n^2) * n * (n^2) = n^5$ so $O(n^5)$. The function is recursively called 3 times and since it is divided by 3 the time complexity of that is log3(n). The total complexity is $O(log3(n)*n^5)$
```

Comment:
Please show the recurrence relation and how you solved it. Your result at the moment is incorrect.